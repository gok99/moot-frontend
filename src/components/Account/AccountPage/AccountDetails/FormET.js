import React from 'react';
import { Col, Row, Form } from 'react-bootstrap';
 
import '../../../Styles/styles.css';
import '../../account.css';

const FormET = (props) => {
  const email = props.email;
  const teleUser = "@" + props.teleUser.toString();
  
  return (
    <Row>
      <Col>
        <Row>
          <p className="text-account">Email</p>
        </Row>
        <Row>
          <Form>
            <Form.Group className="input-account-disabled mt-2" controlId="email">
              <Form.Control
                name="email"
                type="text"
                className="input-text-account"
                value={email}
                disabled>
              </Form.Control>
            </Form.Group>
          </Form>
        </Row>
      </Col>
      <Col>
        <Row>
          <p className="text-account">Telegram Handle</p>
        </Row>
        <Row>
          <Form>
            <Form.Group className="input-account-disabled mt-2" controlId="teleUser">
              <Form.Control
                name="teleUser"
                type="text"
                className="input-text-account"
                value={props.teleUser === '' ? '' : teleUser}
                disabled>
              </Form.Control>
            </Form.Group>
          </Form>
        </Row>
      </Col>
    </Row>
  );
};

export default FormET;