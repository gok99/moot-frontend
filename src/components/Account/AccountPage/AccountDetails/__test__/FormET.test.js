import React from 'react';
import ReactDOM from 'react-dom';
import FormET from '../FormET';

import { cleanup } from '@testing-library/react';
import "@testing-library/jest-dom/extend-expect"
import renderer from 'react-test-renderer';

afterEach(cleanup);

it("renders without crashing", () => {
  const div = document.createElement('div');
  ReactDOM.render(<FormET email={""} teleUser={""} />, div);
  ReactDOM.unmountComponentAtNode(div);
});

it("matches snapshot", () => {
  const tree = renderer.create(<FormET email={""} teleUser={""} />).toJSON();
  expect(tree).toMatchSnapshot();
});