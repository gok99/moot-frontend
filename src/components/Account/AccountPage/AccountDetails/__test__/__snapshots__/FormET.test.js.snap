// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="row"
>
  <div
    className="col"
  >
    <div
      className="row"
    >
      <p
        className="text-account"
      >
        Email
      </p>
    </div>
    <div
      className="row"
    >
      <form
        className=""
      >
        <div
          className="input-account-disabled mt-2 form-group"
        >
          <input
            className="input-text-account form-control"
            id="email"
            name="email"
            readOnly={true}
            type="text"
            value=""
          />
        </div>
      </form>
    </div>
  </div>
  <div
    className="col"
  >
    <div
      className="row"
    >
      <p
        className="text-account"
      >
        Telegram Handle
      </p>
    </div>
    <div
      className="row"
    >
      <form
        className=""
      >
        <div
          className="input-account-disabled mt-2 form-group"
        >
          <input
            className="input-text-account form-control"
            id="teleUser"
            name="teleUser"
            readOnly={true}
            type="text"
            value=""
          />
        </div>
      </form>
    </div>
  </div>
</div>
`;
